docs/how-tos.rst

```diff
@@ -0,0 +1,91 @@
+**********************
+How-Tos, Tips & Tricks
+**********************
+
+This chapter contains some practical examples of using ``shiv`` in the real world,
+to give you a better idea how it can fit into your projects
+and their deployment workflows.
+
+
+Shipping Dependency Sets as a Single File
+=========================================
+
+If you have a bunch of Python scripts that are all using the same set of base libraries,
+you can distribute those dependencies in the form of a zipapp,
+i.e. in a single executable file.
+This way, you can have a project for the base libraries
+and other projects for the scripts,
+including scripts written by your end users.
+
+The following example uses ``shiv`` itself for demonstration purposes,
+you'd use a setuptools project defining the needed dependencies
+when applying this ‘for real’.
+
+So, to create your base library release artifact, call
+``shiv`` like this in your project's workdir::
+
+    PROJECT=shiv
+    shiv -p '/usr/bin/python3.6 -IS' -o ~/bin/_lib-$PROJECT .
+
+Note that we do not provide an entry point here, which means this zipapp
+drops into the given Python interpreter and is thus usable *as* an
+interpeter, but with the dependencies of the project added.
+
+The ``-IS`` options in ``-p`` ensure this zipapp runs isolated (for increased security),
+with neither the current working directory
+nor the host's site packages in the Python path.
+
+.. tip::
+
+    Replace the ``.`` at the end of the ``shiv`` call
+    by a literal list of PyPI packages,
+    or load them from a requirements file (``-r lib-requirements.txt``),
+    and you don't need an extra project for getting your base set defined.
```

Comment:
This is a good tip, can you call out the `.` earlier than this tip as well? I can imagine someone reading this doc may gloss right over the `.` on line 28 and be confused when they see this tip, but if you call it out (after the note about not providing an entry point, maybe) the reader will have proper context. Which is also a good opportunity to let them know that any non-shiv arguments/flags get delegated to `pip`.